src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -39,7 +39,7 @@ def dialog_content
         VBox(
           alternative_information,
           create_alternatives_table,
-          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          RichText(Id(:slaves), "<pre>#{@alternative.choices.first.slaves}</pre>"),
```

Comment:
But even though, it's a bad idea to set the content to "<pre>#{whatever.slaves}</pre>" in two completely different places. You should be reusing a single method to update the widget, called when opening the dialog and also when changing the current selection.